home/home2.1/test_script.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+g++ -o main main.cpp
+
+run_test() {
+    local expected="$1"
+    shift
+    local output=$(./main "$@")
+    if [ "$output" == "$expected" ]; then
+        echo "Test passed: $@"
+    else
+        echo "Test failed: $@"
```

Comment:
Just a note - usually different frameworks make some caps locked words like [FAIL] or something like that so it will be easier to find the fail.